### PR TITLE
Dịch "word2vec-pretraining_vn" - Phần 1

### LINKED ISSUE #2457: Dịch "word2vec-pretraining_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_natural-language-processing-pretraining/word2vec-pretraining_vn.md#L1-L91

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2457 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_natural-language-processing-pretraining/word2vec-pretraining_vn.md [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-*dịch đoạn phía trên*
+Trong phần này, ta sẽ huấn luyện một mô hình skip-gram đã được định nghĩa ở :numref:`sec_word2vec`.
 
 
 <!--
 First, import the packages and modules required for the experiment, and load the PTB dataset.
 -->
 
-*dịch đoạn phía trên*
+Đầu tiên, ta nhập các gói thư viện và mô-đun cần thiết cho thí nghiệm, và nạp dữ liệu PTB.
```
Comment (lkhphuc) [inline chapter_natural-language-processing-pretraining/word2vec-pretraining_vn.md:23]:
```suggestion
Đầu tiên, ta nhập các gói thư viện và mô-đun cần thiết cho thí nghiệm, và nạp tập dữ liệu PTB. 
```

File: chapter_natural-language-processing-pretraining/word2vec-pretraining_vn.md [hunk 2]
```diff
@@ -63,7 +64,9 @@ The weight of the embedding layer is a matrix whose number of rows is the dictio
 We set the dictionary size to $20$ and the word vector dimension to $4$.
 -->
 
-*dịch đoạn phía trên*
+Tầng này được gọi là tầng embedding, trong đó các từ thu được bằng cách phương pháp embedding. Ta sử dụng tầng này bằng cách tạo một thực thể `nn.Embedding` trong Gluon.
```
Comment (lkhphuc) [inline chapter_natural-language-processing-pretraining/word2vec-pretraining_vn.md:67]:
```suggestion
Để thu được các từ bằng cách phương pháp embedding, ta sử dụng tầng embedding bằng cách tạo một thực thể `nn.Embedding` trong Gluon.
```

**Review by lkhphuc — APPROVED** [review]

**Review by duythanhvn — APPROVED** [review]
